listenbrainz/__init__.py

```diff
@@ -4,3 +4,21 @@
 
 DUMP_LICENSE_FILE_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)),
                                       'db', 'licenses', 'COPYING-PublicDomain')
+
+from listenbrainz import default_config as config
+try:
+    from listenbrainz import custom_config as config
+except ImportError:
+    pass
+from brainzutils import cache
+import logging
+
+try:
```

Comment:
doing this here is weird, i think adding a function called `init_cache` in `listenbrainz.utils` and calling it individually in `webserver/__init__.py` (and other places) would be better.